uint16 initial_tick

uint16 final_tick

uint16 frequency

uint16 tick_step

uint16 slave_id








